src/tutorials-web/downloading-fraud-data.md

```diff
@@ -5,19 +5,18 @@ To download fraud event data via FIB Web App, perform the following steps:
 1. On the [Dashboard](../overview/web-interface.md#dashboard) screen, select **Download fraud data** from the [Download](../overview/web-interface.md#download) tab.
 2. On the **Download fraud data** screen that appears, specify the following information about the fraud event:
 
-   - **Fraud type** — select the fraud [type of the fraud events](../overview/fraud-events.md#types-of-fraud-events) that you wish to download.\
+   - **Select fraud data** — select the fraud [type of the fraud events](../overview/fraud-events.md#types-of-fraud-events) that you wish to download.\
```

Comment:
Select should be dropped on all entries, to be adjusted in the app; already done in Figma